runners/google-cloud-dataflow-java/build.gradle

```diff
@@ -319,7 +321,6 @@ task validatesRunnerStreaming {
       'org.apache.beam.sdk.testing.UsesMapState',
       'org.apache.beam.sdk.testing.UsesRequiresTimeSortedInput',
       'org.apache.beam.sdk.testing.UsesSetState',
-      'org.apache.beam.sdk.testing.UsesStrictTimerOrdering',
     ],
```

Comment:
I think this is fine since this inherits from the other tuple, but please double check that this test remains excluded from streaming even after removing this line

Reply:
Ya it's fine. The `validatesRunnerStreaming` task uses the `createLegacyWorkerValidatesRunnerTest` method to build the task which iterates through `commonLegacyExcludeCategories` and adds all of those tests to the exclude pattern. Both batch and streaming use that method, there is a similar one for runnerv2.